threescale/api_test.go

```diff
@@ -38,7 +39,7 @@ func TestClient_Authorize(t *testing.T) {
 		{
 			name:         "Test expect failure bad url passed",
 			auth:         ClientAuth{Type: ProviderKey, Value: "any"},
-			transaction:  &Transaction{Params: Params{AppID: "any"}},
+			transaction:  Transaction{Params: Params{AppID: "any"}},
```

Comment:
Any test with a completely empty transaction meaning no transactions for the authorization call in order to retrieve the current limits state?

Reply:
Yes covered here https://github.com/3scale/3scale-go-client/pull/23/files#diff-1f667ddd6c5e148c86dc54aa68c5147aR200-R229.

I'll verify that we dont generate usage data in said test.